src/mame/sega/model2.cpp

```diff
@@ -3886,6 +3886,60 @@ ROM_START( manxttc ) /* Manx TT Superbike Twin Revision C, Model 2A */
 	MODEL2A_VID_BOARD
 ROM_END
 
+/* This set was badly upgraded to Revision C, lefting over the sound CPU ROM from Revision A.
```

Comment:
Thanks for the PR!
I believe this should be 'leaving' instead of 'lefting'

Reply:
Ooops, fixed, thanks!!!